90_mender_boot_grub.cfg

```diff
@@ -29,10 +29,10 @@ fi
 
 if [ "${drop_to_grub_prompt}" = "no" ]; then
     cryptomount ${mender_grub_storage_device},gpt${mender_boot_part}
-    mender_kernel_path="(crypto0)/boot"
-    if linux ${mender_kernel_path}/${kernel_imagetype} luks.name=c89dc933-90a7-43d1-b3d1-ceecd2e281ee=rootfs root=/dev/mapper/rootfs ${bootargs}; then
-        if test -n "${initrd_imagetype}" -a test -e ${mender_kernel_path}/${initrd_imagetype}; then
-            initrd ${mender_kernel_path}/${initrd_imagetype}
+    mender_kernel_path_crypto="(crypto0)/boot"
```

Comment:
Why not the original `mender_kernel_path`? I can't see it being used after this.

Reply:
I have some strange issues that grub report syntax error (even when script was checked with `grub-script-check`) so this was kind of hack ;). Will be removed.